lib/awful/widget/prompt.lua

```diff
@@ -5,6 +5,7 @@
 --
 -- @author Julien Danjou &lt;julien@danjou.info&gt;
 -- @copyright 2009 Julien Danjou
+-- @copyright 2018 Aire-One
```

Comment:
@awesomeWM/write-access 1) i was always wondering what the difference between author and copyright in this context
2) if we need that for documentation purpose, couldn't it be done while generating documentation based on file history? currently most of the files having those fields mostly out-of-date

Reply:
I added myself as a habit but it actually doesn't mean much. I think it can be interpreted as a "I wrote some part of code here".

About the difference between author and copyright, the ldoc documentation does not provide any hint and I am probably wrong in the use.

Anyway, I doesn't care about it and the statement can be changed/removed (I thought I said it in the PR's first message but I did forget...).

Reply:
By law (at least here), the Copyright is assigned to the employer while the author is an human being. Adding `@copyright` is not, by law, necessary since unless contractually specified otherwise, each contributor to an Open Source project is a contractor. Contractors retain the copyright unless contractually specified (CLAs). So each contributor has an implicit right to the module copyright as long as the code is tinted by his or her contribution.

Reply:
tl;dr: so it's optional?

Reply:
It's at the contributor discretion